app/views/shared/_value_heading.html.erb

```diff
@@ -5,7 +5,7 @@
     <% if value.is_a?(Money) %>
       <span class="<%= size == "lg" ? "text-xl" : "text-lg" %> font-medium"><%= format_money value %></span>
     <% else %>
-      <span class="<%= size == "lg" ? "text-xl" : "text-lg" %> font-medium"><%= is_percentage ? number_to_percentage(value, precision: 2) : value %></span>
+      <span class="<%= size == "lg" ? "text-xl" : "text-lg" %> font-medium"><%= is_percentage ? number_to_percentage(value*100, precision: 2) : value %></span>
```

Comment:
@MrTob I don't think this is the right spot to change this.  This partial is used across _all_ metrics, so while it will fix the "savings rate" metric, it will break the other metrics.

Where I think we need to update this is directly in the query to make sure that we're multiplying the percent fraction *100:

https://github.com/maybe-finance/maybe/blob/195ec85d96bd578098c8dc9ea084de4db28be03b/app/models/family.rb#L158